src/database.c

```diff
@@ -97,13 +101,17 @@ log_database_init(ProfAccount* account)
     }
 
     char* err_msg;
-    // id is the ID of DB the entry
-    // from_jid is the senders jid
-    // to_jid is the receivers jid
-    // from_resource is the senders resource
-    // to_jid is the receivers resource
-    // message is the message text
-    // timestamp the timestamp like "2020/03/24 11:12:14"
+
+    // ChatLogs Table (Version 1)
```

Comment:
Why would we always want to start off with `version 1`? Wouldn't it make more sense to check the version first, if we're at `none` or `<latest>` we're fine and directly create `version <latest>` style tables.

If we're at `version <not latest>` we start the migration.

Reply:
That's a common practice with migrations, since not only the DB structure changes, but also triggers, indexes are added etc. Totally it would be a big DRY violation and it would just make all new migrations harder, since changes would need to be duplicated.